onnxmltools/convert/sklearn/operator_converters/KNN.py

```diff
@@ -5,10 +5,18 @@
 # --------------------------------------------------------------------------
 
 from ....proto import onnx_proto
-from ...common._apply_operation import apply_abs, apply_mul, apply_reshape, apply_sub
+from ...common._apply_operation import apply_abs, apply_mul, apply_reshape, apply_sub, apply_cast
```

Comment:
Please sort `apply_` by alphabets so that `apply_case` would be the 2nd imported function.